tests/phpunit/elementor/includes/template-library/test-manager-cloud.php

```diff
@@ -119,4 +119,60 @@ public function test_save_template() {
 			'parentId' => null,
 		]);
 	}
+
+	public function test_load_more_templates() {
+		// Arrange
+		$args = [
+			'offset' => 10,
+			'search' => 'search',
+			'source' => 'cloud',
+		];
+
+		$this->cloud_library_app_mock
+			->expects( $this->once() )
+			->method('get_resources')
+			->with($args);
+		// Act
+		$this->manager->load_more_templates($args);
+	}
+
+	public function test_load_more_templates_fails_wuthout_source() {
+		// Arrange
+		$args = [
+			'search' => 'search',
+			'offset' => 10,
+		];
+
+		// Act
+		$result = $this->manager->load_more_templates($args);
+
+		// Assert
+		$this->cloud_library_app_mock
+			->expects( $this->never() )
+			->method('get_resources');
+
+		$this->assertWPError( $result );
+
+		$this->assertEquals( 'The required argument(s) "source" not specified.', $result->get_error_message() );
+	}
+
+	public function test_load_more_templates_fails_wuthout_offset() {
+		// Arrange
+		$args = [
+			'search' => 'search',
+			'source' => 'cloud',
+		];
+
+		// Act
+		$result = $this->manager->load_more_templates($args);
```

Comment:
```suggestion
		$result = $this->manager->load_more_templates( $args );
```
Check spacing in the file